accounting/fiscality/taxes/taxes.rst

```diff
@@ -0,0 +1,273 @@
+
+  For example, a product has a Sales Price of $1000, and we apply a *10% of Price* tax. We then
+  have:
+
+  +-------------+-------------+----------+----------+
+  | Product's   | Price       | Tax      | Total    |
+  | Sales Price | without tax |          |          |
+  +=============+=============+==========+==========+
+  | 1,000       | 1,000       | 100      | 1,100.00 |
+  +-------------+-------------+----------+----------+
+
+- **Percentage of Price Tax Included**
+
+  The *Total* is the taxable basis: the tax's amount is a percentage of the Total.
+
+  For example, a product has a Sales Price of $1000, and we apply a *10% of Price Tax Included* tax.
+  We then have:
+
+  +-------------+-------------+----------+----------+
+  | Product's   | Price       | Tax      | Total    |
+  | Sales Price | without tax |          |          |
+  +=============+=============+==========+==========+
+  | 1,000       | 1,000       | 111.11   | 1,111.11 |
+  +-------------+-------------+----------+----------+
+
+.. _taxes/active:
+
+Active
+~~~~~~
+
+Only **active** taxes can be added to new documents.
+
+.. important::
+   It is not possible to delete taxes that have been used to record an entry. Instead, you can
+   deactivate these taxes to prevent users from using them.
+
+.. note::
+   This field can be modified from the *List View*. See :ref:`above <taxes/list_activation>` for
+   more information.
+
+.. _taxes/scope:
+
+Tax Scope
+~~~~~~~~~
+
+The **Tax Scope** determines the tax's application and where it is displayed.
+
+- **Sales**
+- **Purchase**
+- **None**
+
+.. tip::
+   You can use **none** for taxes that you want to include in a :ref:`Group of Taxes
+   <taxes/computation>` but that you don't want to list along with other Sales or Purchase taxes.
+
+.. _taxes/definition-tab:
+
+Definition tab
+--------------
+
+Allocate with precision the amount of the taxable basis or percentages of the computed tax to
+multiple accounts and Tax Grids.
+
+.. image:: media/taxes-definition.png
+   :align: center
+   :alt: Allocate tax amounts to the right accounts and tax grids
+
+- **Based On**:
+
+  - Base: The price on the invoice line
+  - % of tax: a percentage of the computed tax.
+
+- **Account**: if defined, an additional Journal Item is recorded.
+- **Tax Grids**: Tax Grids are used to generate  your :doc:`Tax Reports
+  <../../reporting/declarations/tax_returns>` automatically, according to your country's regulations.
+
+.. _taxes/advanced-tab:
+
+Advanced Options tab
+--------------------
+
+.. _taxes/label-invoices:
+
+Label on Invoices
+~~~~~~~~~~~~~~~~~
+
+The label of the tax, as displayed on each invoice line in the **Taxes** column.
+
+.. image:: media/taxes-invoice-label.png
+   :align: center
+   :alt: The Label on Invoices is displayed on each invoice line
+
+.. important::
+   The label displayed between the Subtotal and Total lines is defined by the :ref:`Tax Group
+   <taxes/tax-group>` name and *not* by the Label on Invoices.
```

Comment:
No it's good you're pointing this out.
But in the section above I write this about Tax Names:

```
Tax Name
~~~~~~~~

The **Tax Name** as you want to display it on Odoo.

.. note::
   To modify how the tax is labeled on invoices, please see below: :ref:`taxes/label-invoices`.
```

So your test might have had the same tax name and label, but the tax name is only in backend :)

Reply:
And indeed, the way it works IS confusing. It tool me some while to figure it out correctly...
The kind of things that is _really_ time-consuming ^^

Reply:
@kbr-odoo 

I'm adding this in the "Configuration" section, with a reference to it in the other sections. What do you think?


.. _taxes/appellations:

```
.. important::
   Taxes have three different appellations, altough each one is linked to a different field with a 
   specific use. Refer to the following table to see where they are displayed.

   +----------------+-------------------------+----------------------+
   | :ref:`Tax Name | :ref:`Label on Invoice  | :ref:`Tax Group      |
   | <taxes/name>`  | <taxes/label-invoices>` | <taxes/tax-group>`   |
   +================+=========================+======================+
   | Backend        | *Taxes* column          | Above the *Total*    |
   |                | on invoices             | line on invoices     |
   +----------------+-------------------------+----------------------+

```

Reply:
But I did try with different names, and the name I set on *Label on Invoice* was not the one used. I am doing something wrong then: https://nimb.ws/Lm3aBW.

I believe the doc might have too many ..notes and ..important sections about it now. Once it is explained under the title, like it was before, it might be enough. Otherwise, the doc also becomes longer, which might give the impression of difficulty, something we want to avoid.
:)

suggestion: appelations -> names

Reply:
@kbr-odoo 
No worries :)
But you are still in back end.
I you click on preview, or print the invoice, or see it on the customer portal, it is the Label on Invoice that is displayed.
The tax name is only for you as a back end user.

Reply:
got it.. thank you, Jon ^^

Reply:
suggestions to consider:

- Not have this information repeated many times (keep it like it was in your first version). Maybe adding the word *Customer* before *Invoice* + more contextualized screenshots.
- suggestion: appelations -> names

o/o/o/

Reply:
Last commit with all your suggested recommendations (although I choose "labels" instead of "names" to avoid confusion in the texts).